src/utils/getDomainName.ts

```diff
@@ -0,0 +1,2 @@
+export const getDomainName = (url: string) =>
+	url?.startsWith("http://") || url.startsWith("https://") ? url.split("//")[1].replace("/", "") : url;
```

Comment:
it should be safe to remove the last trailing slash. Something like this:
```
if (url.endsWith('/')) {
        return url.slice(0, -1);
    }
    return url;
```

Reply:
@vinguthh 
That's what this is doing 
`url.split("//")[1].replace("/", "")`

Reply:
@vinguthh updated the condition, found an edge case. Resolved

Reply:
But this assume that the url in this type of format: `http://something/`, then we will get `http://something`. If the url is `http://something/somethingelse`, then we will get `http://somethingsomethingelse`. I found it's safe to not assume the format of url but just to remove the last trailing slash.